app/assets/stylesheets/colors.scss

```diff
@@ -0,0 +1,5 @@
+$dropbox-blue: #007ee5;
+$white: #FFFFFF;
+$lighter-cool-grey: #7b8994;
+$light-cool-grey: #47525d;
+$light-dark-cool-grey: #3d464d;
```

Comment:
Love that these are split out into a different sheet and that you are using variables.